schema/dpns-contract-documents.json

```diff
@@ -82,8 +82,7 @@
         "properties": {
           "allowSubdomains": {
             "type": "boolean",
-            "const": false,
-            "description": "This option defines who can create subdomains. Only domain owner is allowed at the moment"
+            "description": "This option defines who can create subdomains"
```

Comment:
@jawid-h @antouhou I would add a description what `true` and `false` does mean and also add a comment to explain that `true` is only for TLD atm.